FableAnniversary.asl

```diff
@@ -42,6 +45,13 @@ startup {
 	}
 	vars.inLoad = false;
 }
+start {
+	if(current.startingX >= 3494 && current.startingX <= 3495 && current.startingZ >= 867 && current.startingZ <= 868){

```

Comment:
This is where you spawn into oakvale?  What are the common expected values?

Reply:
yes the expected values should be x-axis 3494.918457 and the z-axis 867.8829346. I couldn't get it to take those exact coordinates to save my life, so I had it checked for anything in between 3494-3495 and 867-868.

Reply:
Spoke with Ligitacy about this; my recommendation is to check >=3494 and <= 3496, same for 867-869